src/status_im/contexts/chat/group_details/view.cljs

```diff
@@ -166,4 +168,15 @@
        :render-section-footer-fn       contacts-section-footer
        :render-data                    {:chat-id chat-id
                                         :admin?  admin?}
-       :render-fn                      contact-item-render}]]))
+       :render-fn                      contact-item-render}]
+     [quo/floating-shell-button
```

Comment:
hey @Parveshdhull instead of adding this to each screen do you think it would be possible just to have `:shell-button? true` option for the screen?

Reply:
agree with @flexsurfer here, should be a generic solution where possible 👍

Reply:
Created an issue: https://github.com/status-im/status-mobile/issues/19192